lib/miq_pglogical.rb

```diff
@@ -80,10 +93,8 @@ def self.save_global_region(subscriptions_to_save, subscriptions_to_remove)
     PglogicalSubscription.save_all!(subscriptions_to_save)
   end
 
-  private
-
-  def pglogical(refresh = false)
-    @pglogical = nil if refresh
-    @pglogical ||= PG::LogicalReplication::Client.new(@pg_connection)
+  def self.pg_connection
+    ApplicationRecord.connection.raw_connection
```

Comment:
Again, just a slightly different implementation than what is done in `PglogicalSubscription`.